docs/source/index.rst

```diff
@@ -8,30 +8,192 @@ ceph-ansible
 Ansible playbooks for Ceph, the distributed filesystem.
 
 
-Testing
-=======
+Installation
+============
 
-* :doc:`Testing with ceph-ansible <testing/index>`
-* :doc:`Glossary <testing/glossary>`
+github
+------
+You can install directly from the source on github by following these steps:
+
+- Clone the repository::
+
+      git clone https://github.com/ceph/ceph-ansible.git
+
+- Next, you must decide which branch of ``ceph-ansible`` you wish to use. There
+  are stable branches to choose from or you could use the master branch::
+
+      git checkout $branch
+
+Releases
+========
+
+The following branches should be used depending on your requirements. The ``stable-*``
+branches have been QE tested and sometimes recieve backport fixes throughout their lifecycle.
+The ``master`` branch should be considered experimental and used with caution.
+
+- ``stable-2.1`` Support for ceph version ``jewel``. This branch supports ansible versions
+  ``2.1`` and ``2.2.1``.
+
+- ``stable-2.2`` Support for ceph versions ``jewel`` and ``kraken``. This branch supports ansible versions
+  ``2.1`` and ``2.2.2``.
+
+- ``master`` Support for ceph versions ``jewel``, ``kraken`` and ``luminous``. This branch supports ansible versions
+  ``2.2.3`` and ``2.3.1``.
+
+Configuration and Usage
+=======================
+This project assumes you have a basic knowledge of how ansible works and have already prepared your hosts for
+configuration by ansible.
+
+After you've cloned the ``ceph-ansible`` repository, selected your branch and installed ansible then you'll need to create
+your inventory file, playbook and configuration for your ceph cluster.
+
+Inventory
+---------
+
+The ansible inventory file defines the hosts in your cluster and what roles each host plays in your ceph cluster. The default
+location for an inventory file is ``/etc/ansible/hosts`` but this file can be placed anywhere and used with the ``-i`` flag of
+ansible-playbook. An example inventory file would look like::
+
+    [mons]
+    mon1
+    mon2
+    mon3
+
+    [osds]
+    osd1
+    osd2
+    osd3
+
+.. note::
+
+    For more information on ansible inventories please refer to the ansible documentation: http://docs.ansible.com/ansible/latest/intro_inventory.html
+
+Playbook
+--------
+
+You must have a playbook to pass to the ``ansible-playbook`` command when deploying your cluster. There is a sample playbook at the root of the ``ceph-ansible``
+project called ``site.yml.sample``. This playbook should work fine for most usages, but it does include by default every daemon group which might not be
```

Comment:
This part is where i disagree, I think it should be used as is to reduce any user/admin errors or if user runs his own custom playbook it could be considered unsupported or something?